app_sensor_module/src/main.c

```diff
@@ -26,6 +27,41 @@ static const struct gpio_dt_spec led0 = GPIO_DT_SPEC_GET(LED0_NODE, gpios);
 static const struct gpio_dt_spec led1 = GPIO_DT_SPEC_GET(LED1_NODE, gpios);
 static const struct device *const wiznet = DEVICE_DT_GET_ONE(wiznet_w5500);
 
+static void ina_task(void *, void *, void *) {
+    struct sensor_value temperature;
+    power_module_telemetry_t sensor_telemetry = {0};
+    
+    const struct device *tmp116_device = DEVICE_DT_GET_ANY(ti_tmp116);
+    if (!device_is_ready(tmp116_device)) {
+        LOG_ERR("whomp whomp TMP116 sensor not found");
```

Comment:
Its fine to meme but try to keep the logging professional in the final version of the code